src/app/register/Validators.ts

```diff
@@ -0,0 +1,10 @@
+import { AbstractControl } from '@angular/forms';
+
+export const ConfirmPasswordValidator = (control: AbstractControl): { [key: string]: any} => {
+  const value = control.get('password').value;
+  const otherValue = control.get('confirmpassword').value;
+  if (value !== otherValue) {
+    return {doNotMatch: {value, otherValue}};
+  }
+  return null;
+};
```

Comment:
new line

Reply:
Done.

Reply:
@lunayach Configure your editor to do this when ever you save. It was possible in atom, must be possible with Visual Studio Code.

Reply:
Ya! there was an option and I checked it. So any new files that are in newer branches than this one, should not have the problem.